cmd/jag/commands/device.go

```diff
@@ -73,15 +73,15 @@ func (d Device) Ping(ctx context.Context, sdk *SDK) bool {
 	return res.StatusCode == http.StatusOK
 }
 
-func (d Device) Run(ctx context.Context, sdk *SDK, b []byte, options string) error {
+func (d Device) Run(ctx context.Context, sdk *SDK, b []byte, defines string) error {
 	req, err := http.NewRequestWithContext(ctx, "PUT", d.Address+"/code", bytes.NewReader(b))
 	if err != nil {
 		return err
 	}
 	req.Header.Set(JaguarDeviceIDHeader, d.ID)
 	req.Header.Set(JaguarSDKVersionHeader, sdk.Version)
-	if len(options) > 0 {
-		req.Header.Set(JaguarRunOptionsHeader, options)
+	if len(defines) > 0 {
```

Comment:
Personally I prefer to check against "":
```suggestion
	if defines != "" {
```
Fine either way, though.

Reply:
Done.